shared/src/main/java/com/faforever/neroxis/mask/FloatMask.java

```diff
@@ -237,7 +287,11 @@ public FloatMask addGaussianNoise(float scale) {
      * @param scale Multiplicative factor for the noise
      */
     public FloatMask addWhiteNoise(float scale) {
-        return addPrimitiveWithSymmetry(SymmetryType.SPAWN, (x, y) -> random.nextFloat() * scale);
+        if (symmetrySettings.getSymmetry(SymmetryType.SPAWN).isOddSymmetry()) {
```

Comment:
Before:
![image](https://github.com/FAForever/Neroxis-Map-Generator/assets/11024011/fc74d82c-83fe-49fe-9f7f-2bef91517071)

After:
![image](https://github.com/FAForever/Neroxis-Map-Generator/assets/11024011/dc191351-1175-4cf0-a13e-63baede5e7cf)

I tried to figure how to make this perfect symmetry without any artifacting, but found it impossible.. lol..
So I kinda gave up... and just made it look uniform.

Reply:
Unfortunately the rest of the code and generation relies on the symmetry being maintained so just a uniform distribution is not good enough.